modules/nlu/src/backend/pipelines/language/none-generator.ts

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash'
+
+export const SEPARATOR_CHAR = ''
+
+// Example input: [ [ 'he', 'llo', 'what', 'is', 'your', 'name', '?' ] ]
+// Example output: [ [ 'haatlo', 'llmeour', 'your', 'name', 'ame', 'atyona', 'he' ], [ 'amheur', 'am', 'llo', 'is' ] ]
+export function generateNoneUtterances(utterances: string[][], count: number): string[][] {
```

Comment:
We should take in consideration tfidf and only use words that are low rated as real tokens